doc/changes/devel/13019.newfeature.rst

```diff
@@ -0,0 +1 @@
+Add ``fig.mne`` container for :class:`Colorbar <matplotlib.colorbar.Colorbar>` in :func:`plot_connectivity_circle <mne_connectivity.viz.plot_connectivity_circle>` to allow users to access it directly, by `Santeri Ruuskanen`_.
```

Comment:
FYI for future reference I think this would have been equivalent and shorter!
```suggestion
Add ``fig.mne`` container for :class:`~matplotlib.colorbar.Colorbar` in :func:`plot_connectivity_circle <mne_connectivity.viz.plot_connectivity_circle>` to allow users to access it directly, by `Santeri Ruuskanen`_.
```